lib/assessment.js

```diff
@@ -4,23 +4,39 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+var min = function () {
+  var minimum = arguments[0]
+  for (i=1; i< arguments.length; i++){
+      if (arguments[i] < minimum) {
+        mimumun = arguments][i];
+    }
+  }
+  return minimum;
+}
 
-};
+// I know this is wrong, I need to put args in min(args) but cant remember how to set it to the next variable because when I type in arg = arguments, arugments is already defined and my code will only accept three cases right now because of how its set up as a simple if/then statement.
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
-
+var sum = function sum(arg1,arg2) {
+    var total = 0;
+    for (var i = 0; i < arguments.length; i++){
+      total += arguments[i];
+    }
+    return total;
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
-
+  var array = [];
+  for (var i = 0; i < arguments.length; i++){
```

Comment:
This loop is counting up to the wrong number. There is a `length` parameter to this function that is intended to tell it how long to make the array. However, your loop that populates the array counts up to the number of arguments passed in. Under correct usage, this function would always produce an array of length 2.